docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,363 @@
+
+Online Certificate Status Protocol (OCSP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the configuration of OCSP, the following changes are necessary.
+
+First we need to allow the use of the template `OCSPResponseSigning` by the server, to do so navigate to `Server Manager -> Roles -> Active Directory Certificates Services`, expand `YourServerName`, right click `Certificate template` and choose `New template to issue`, in the list select `OCSPResponseSigning`.
+
+After the installation of OCSP we need to create a Revocation Configuration.
+
+To create the Revocation Configuration navigate to `Server Manager -> Roles -> Active Directory Certificate Services` and expand `OnlineResponder`. Then expand `YourServerName`, right click on Revocation Configuration, select `Add Revocation Configuration`, click `Next`, choose a name for your configuration and click `Next`. 
+
+Choose `Select a certificate for an existing entreprise CA`, click `Next`. Click `Browse` and find your entreprise CA in the list, select it, click `OK` and then `Next`. Choose `Automatically select a signing certificate`, make sure `Auto-Enroll for an OCSP signing certificate` is selected, then choose the default template of OCSP which is `OCSPResponseSigning` in the drop down list next to `Certificate Template:`. You need to add providers only if you wish to use a CRL in addition to OCSP.
+
+Once created, right click the revocation configuration and select `Edit properties`, go to the `Signing` tab, then select `Enable NONCE extension support` then click `OK`.
+
+Make sure that your OCSP server appear in the CA settings. Right click your CA, choose `Properties`. Navigate to the tab `Extension`, in the drop down list `Select extension` choose `Authority Information Access (AIA)`. Make sure that you have the following in the list of locations: `http://<ServerDNSName>/OCSP`. 
+
+If you do not have it, add it via the button `Add...`. In this menu type the `http://` then insert `<ServerDNSName>` and type `/OCSP`, then validate by clicking `OK`. Also verify that `Include in the online certificate status protocol(OCSP) extension` is selected.
+
+By default OCSP has a two days delay to refresh it's CRL information. Which means if you revoke a certificate on MSPKI, it will take two days before PacketFence detects the certificate is revoked. If this delay is too long for your needs, you can change it on the NDES server. To do so, navigate to `Server Manager -> Roles -> Active Directory Certificate Service` and right click `Entreprise PKI`, in the menu select `Options...`. The delay can be changed by modifying the value of `Set CRL status to Expiring when expiring in:` to your convenience.
+
+
+RADIUS Certificate generation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the PKI of Microsoft include that all your certificates will be deliver by the root CA of MSPKI. The public key of this certificate is necessary on PacketFence. 
```

Comment:
Using the Microsoft PKI involves that all your certificates will be delivered by the root CA of the MSPKI.